internal/cmd/server.go

```diff
@@ -73,7 +73,7 @@ func newServerCmd() *cobra.Command {
 	cmd.Flags().String("db-encryption-key", "", "Database encryption key")
 	cmd.Flags().String("db-encryption-key-provider", "", "Database encryption key provider")
 	cmd.Flags().Bool("enable-telemetry", false, "Enable telemetry")
-	cmd.Flags().BoolVar(&options.UI.Enabled, "enable-ui", false, "Enable Infra server UI")
```

Comment:
Keeping this option and hiding it isn't necessary since it's not something that should be used currently